lib/patches/tenant_worker.rb

```diff
@@ -0,0 +1,12 @@
+require 'sidekiq'
+
+class Patches::TenantWorker
+  include Sidekiq::Worker
+  include Patches::TenantRunConcern
+
+  sidekiq_options Patches::Config.configuration.sidekiq_options
```

Comment:
Can this just be added to `Patches::TenantRunner`?

Reply:
We need to have a new job enqueued for each tenant.

Reply:
just learnt that we can configure `sidekiq_options`. Couldn't find it anywhere in the docs, is it worth mentioning it?

Reply:
good idea, added to the note in the usage instructions.